ADR/0044-spdx-support.md

```diff
@@ -0,0 +1,302 @@
+      },
+      {
+          "spdxElementId": "SPDXRef-DocumentRoot-Directory-.",
+          "relatedSpdxElement": "SPDXRef-Package-python-attrs-eef51168ca2a575f",
+          "relationshipType": "CONTAINS"
+      }
+      ...
+  ]
+}
+```
+
+And we want that to express that SBOM is actually generated for a container image not for source directory.
+So we remove `SPDXRef-DocumentRoot-Directory-.` package and add new virtual package representing the container image. And replace SPDX ID in relationships with ID of the new package. New SBOM should look like this:
+
+```jsonc
+{
+  "SPDXID": "SPDXRef-DOCUMENT",
+  ...
+  "packages": [
+      {
+        "name": "my-image",
+        "SPDXID": "SPDXRef-image",
+        "versionInfo": "latest",
+        ...
+        "checksums": [
+            {
+                "algorithm": "SHA-256",
+                "checksumValue": "9ac75c1a392429b4a087971cdf9190ec42a854a169b6835bc9e25eecaf851258"
+            }
+        ],
+        ...
+        "externalRefs": [
+            {
+                "referenceCategory": "PACKAGE-MANAGER",
+                "referenceType": "purl",
+                "referenceLocator": "pkg:oci/my-image@sha256:9ac75c1a392429b4a087971cdf9190ec42a854a169b6835bc9e25eecaf851258?repository_url=container-registry.com/my-org/my-image"
+            }
+        ],
+        "primaryPackagePurpose": "CONTAINER"
+      },
+      {
+      {
+        "name": "attrs",
+        "SPDXID": "SPDXRef-Package-python-attrs-eef51168ca2a575f",
+        "versionInfo": "24.2.0",
+        ...
+      }
+      ...
+  ],
+  "relationships": [
+      {
+        "spdxElementId": "SPDXRef-DOCUMENT",
+        "relatedSpdxElement": "SPDXRef-DocumentRoot-Image-container-registry.com/my-org/my-image:latest",
+        "relationshipType": "DESCRIBES"
+      },
+      {
+          "spdxElementId": "SPDXRef-DocumentRoot-Image-container-registry.com/my-org/my-image:latest",
+          "relatedSpdxElement": "SPDXRef-Package-python-attrs-eef51168ca2a575f",
+          "relationshipType": "CONTAINS"
+      }
+      ...
+  ]
+}
+```
+
+#### SPDX specific attributes
+
+There are SPDX attributes which are required to be present in the document, however there's
+no cycloneDX equivalent for them. These attributes are:
+
+##### Document.documentNamespace
+
+documentNamespace is URI which provides way how locate the document or reference it other documents. When creating SPDX document locally via syft or cachi2, this attribute has no meaning
+as it's not clear yet how the document will be published. Therefore it's set to `NOASSERTION`.
+However when the document is copied to a container. It would make sense to provide a link to the document container
+
+##### Package.downloadLocation
+downloadLocation is URI which provides way how to download the package. When creating SPDX document locally via syft, this information is not available. Also syft doesn't provide anything to this field. Therefore it's set to `NOASSERTION`.
```

Comment:
More general rather than just focusing on Syft please :pray: 
```suggestion
downloadLocation is URI which provides way how to download the package. This is not always available, and it's not clear if it's useful. Therefore it's set to `NOASSERTION`.
```